### PR TITLE
Support style block map

### PR DESCRIPTION
**Pull request by jackyho112**
Support tags for inline styles

### CODE REVIEW

File: spec/integrations/html_spec.rb [hunk 1]
```diff
@@ -45,6 +45,12 @@
       },
       style_map: {
         'ITALIC' => { fontStyle: 'italic' }
+      },
+      style_block_map: {
+        'ITALIC' => 'i',
+        'BOLD' => 'b',
+        'UNDERLINE' => 'u',
+        'STRIKETHROUGH' => 's'
```
Comment (MikeKlemarewski) [inline spec/integrations/html_spec.rb:53]: This seems like something that should be a Class level constant, not passed into the initializer.

File: lib/draftjs_exporter/html.rb [hunk 2]
```diff
@@ -42,13 +43,27 @@ def block_contents(element, block, entity_map)
 
     def add_node(element, text, state)
       document = element.document
+      parent = build_nested_tag_element(state.element_style_tags, element)
```
Comment (MikeKlemarewski) [inline lib/draftjs_exporter/html.rb]: I'm kinda confused how this code works. What is element here, and what is state?

**Comment by jvaill** [comment]
It this going to work with overlapping styles that don't end at the same boundary?

Like this:
```
<b><s><i>hey</i> y</s>o</b>
```
?

**Comment by jvaill** [comment]
Can we make it not block up if you don't include `style_block_map`?

**Comment by jvaill** [comment]
👍

**Pull request merged**